libraries/redcore/api/hal/model/item.php

```diff
@@ -123,7 +125,15 @@ public function getItem($pk = null)
 
 		if (property_exists($item, 'params'))
 		{
-			$registry = new Registry;
+			if (version_compare(JVERSION, '3.3', '<'))
```

Comment:
I think only "use Joomla\Registry\Registry;" is needed since it adds JRegistry as an alias...